packages/constants/src/models.ts

```diff
@@ -69,15 +69,22 @@ export type ProviderLog = {
 
 export type DocumentVersion = {
   id: number
+  createdAt: Date
+  updatedAt: Date
+  deletedAt: Date | null
   documentUuid: string
+  commitId: number
   path: string
   content: string
   resolvedContent: string | null
-  contentHash: string
-  promptlVersion: number
-  commitId: number
+  contentHash: string | null
   datasetId: number | null
```

Comment:
@andresgutgon can we remove this column?

Reply:
Ah true, yes, this is the old one then. Also these models are fake